threading/includes/global_data.h

```diff
@@ -0,0 +1,80 @@
+#ifndef __GLOBAL_DATA_H__
+#define __GLOBAL_DATA_H__
+
+#define _XOPEN_SOURCE 700
+#define _GNU_SOURCE
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <inttypes.h>
+#include <unistd.h>
+#include <pthread.h>
+#include <errno.h>
+
+#include "server.h"
+#include "my_queue.h"
+
+#define SERV_ADDR       "127.0.0.1"
```

Comment:
Don't use hard-coded IP addresses. Panelists don't like it. There are macros in C that specify localhost.